catalog-info.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: zocalo
+  title: Zocalo
+  description: >
+    Infrastructure components for automated data processing via a messaging
+    framework.
+  annotations:
+    github.com/project-slug: DiamondLightSource/python-zocalo
+    diamond.ac.uk/viewdocs-url: https://zocalo.readthedocs.io
+  tags:
+    - python
+spec:
+  type: library
+  lifecycle: production
+  owner: group:data-analysis
+dependsOn:
+  - resource:zocalo-rabbitmq
+  - resource:workflows
```

Comment:
What is this resource? Am I right to assume it has not yet been created?

Reply:
This resource would be created by python-workflows#167 - sorry I should have linked to that PR here

Reply:
Gotcha, must've missed that one in the deluge I've been getting this morning :laughing:

Reply:
`workflows` appears to be a `Component`, not a `resource`, I expect this needs changed